src/ServicePulse.Host/app/js/views/failed_groups/view.html

```diff
@@ -15,9 +15,24 @@
 
         <no-data ng-show="vm.exceptionGroups.length === 0 && !vm.loadingData" title="message groups" message="There are currently no grouped message failures"></no-data>
 
-        <div class="row">
-            <div class="col-sm-12" ng-show="vm.exceptionGroups.length > 0">
+        <div class="row" ng-show="vm.availableClassifiers.length && vm.exceptionGroups.length">
```

Comment:
This condition means that if we switch to classification that has no groups we won't be able to go back, right?

I think we should remove `vm.exceptionGroups.length` part.